src/plugin.ts

```diff
@@ -53,7 +53,7 @@ class CacheMongoose {
   }
 
   public async clear (customKey?: string): Promise<void> {
-    if (customKey) {
+    if (typeof customKey === 'string') {
```

Comment:
If I accidentally pass any type like object, symbol, number that would clear the cache and it would be a disaster 😆.

I think better to check for null or undefined with something like `customkey != null `which will return false for `null` and `undefined`.

Reply:
Hahah true, completely ignored else condition, yep lets do `customkey != null`